python/pdo/test/contract.py

```diff
@@ -389,6 +389,8 @@ def UpdateTheContract(config, enclaves, contract, contract_invoker_keys) :
             update_response = update_request.evaluate()
 
             raw_result = str(update_response.invocation_response)
+            # remove escaped quotes json embeds into string values passed to rsp.error()
+            raw_result = raw_result.replace('"\\"', '').replace('\\""', '')
             result = raw_result[:15]
```

Comment:
for all that i like this for convenience, every time i've tried to "fix" quotes, bad things have happened. in this case, i don't see how any of your tests change as a result of this? could we just leave the raw values (for better or worse) for the moment?

Reply:
I suppose we can. I would love to have a more consistent way of reporting internal PDO exceptions, WASM exceptions, and contract errors back to the client, at least for testing purposes, since some of these returned messages have quotes in quotes and some don't, and some return the actual error message and some return the same message for several different types of errors.